app.json

```diff
@@ -1,7 +1,7 @@
 {
     "id": "bc4dd4a1-bf9b-408e-83a4-aba7eba0bf02",
-    "version": "0.1.0",
-    "requiredApiVersion": "^1.42.2",
+    "version": "0.1.1",
+    "requiredApiVersion": "^1.44.0",
```

Comment:
Do we have this version already? I get an error trying to install here.

Reply:
It's the current alpha version.